src/test/java/duckie/DuckieTest.java

```diff
@@ -0,0 +1,12 @@
+package duckie;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class DuckieTest {
+    @Test
+    public void dummyTest() {
+        assertEquals(2, 2);
+    }
+}
```

Comment:
Could consider getting rid of this test class since it actually is just a dummy test that doesn't really test any of the features in your actual code